docs/licenses.md

```diff
@@ -0,0 +1,298 @@
+* **Open Data**: only accepting [conformant licenses](https://opendefinition.org/licenses/) to the Open Definition from the Open Data Commons.
+* **Open Source Software**: only accepting [approved licenses](https://opensource.org/licenses) from the Open Source Initiative.
+
+The full list of licenses is included below.
+
+## Content
+
+Full Name | SPDX Identifier
+--- | ---
+CC-BY-1.0 | Creative Commons Attribution 1.0 Generic
+CC-BY-2.0 | Creative Commons Attribution 2.0 Generic
+CC-BY-2.5 | Creative Commons Attribution 2.5 Generic
+CC-BY-3.0 | Creative Commons Attribution 3.0 Unported
+CC-BY-3.0-AT | Creative Commons Attribution 3.0 Austria
+CC-BY-4.0 | Creative Commons Attribution 4.0 International
+CC-BY-NC-1.0 | Creative Commons Attribution Non Commercial 1.0 Generic
+CC-BY-NC-2.0 | Creative Commons Attribution Non Commercial 2.0 Generic
+CC-BY-NC-2.5 | Creative Commons Attribution Non Commercial 2.5 Generic
+CC-BY-NC-3.0 | Creative Commons Attribution Non Commercial 3.0 Unported
+CC-BY-NC-4.0 | Creative Commons Attribution Non Commercial 4.0 International
+CC-BY-NC-SA-1.0 | Creative Commons Attribution Non Commercial Share Alike 1.0 Generic
+CC-BY-NC-SA-2.0 | Creative Commons Attribution Non Commercial Share Alike 2.0 Generic
+CC-BY-NC-SA-2.5 | Creative Commons Attribution Non Commercial Share Alike 2.5 Generic
+CC-BY-NC-SA-3.0 | Creative Commons Attribution Non Commercial Share Alike 3.0 Unported
+CC-BY-NC-SA-4.0 | Creative Commons Attribution Non Commercial Share Alike 4.0 International
+CC-BY-SA-1.0 | Creative Commons Attribution Share Alike 1.0 Generic
+CC-BY-SA-2.0 | Creative Commons Attribution Share Alike 2.0 Generic
+CC-BY-SA-2.5 | Creative Commons Attribution Share Alike 2.5 Generic
+CC-BY-SA-3.0 | Creative Commons Attribution Share Alike 3.0 Unported
+CC-BY-SA-3.0-AT | Creative Commons Attribution Share Alike 3.0 Austria
+CC-BY-SA-4.0 | Creative Commons Attribution Share Alike 4.0 International
+CC0-1.0 | Creative Commons Zero v1.0 Universal
+
+## Data
+
+Full Name | SPDX Identifier
+--- | ---
+CC-BY-4.0 | Creative Commons Attribution 4.0 International
+CC-BY-SA-4.0 | Creative Commons Attribution Share-Alike 4.0 International
+CC0-1.0 | Creative Commons Zero v1.0 Universal
+ODbL-1.0 | Open Data Commons Open Database License 1.0
+ODC-By-1.0 | Open Data Commons Attribution License 1.0 
+PDDL-1.0 | Open Data Commons Public Domain Dedication and Licence 1.0
+
+## Software
```

Comment:
I am probably missing some context, but I wonder if there is a good reason not to link out to the OSI's website instead of maintaining this list ourselves? https://opensource.org/licenses/alphabetical

Reply:
Very good observation: we are indeed linking to the list that OSI maintains on their website from the main README, and from the very top of this README. However, there are very small and subtle differences in that the list that OSI maintains is not consistent with the use of SPDX identifiers, for example to name a few:
*  `LiliQ-R+` (OSI) vs `LiLiQ-Rplus-1.1` (SPDX)
* the distinction that SPDX does between `MPL-2.0` and `MPL-2.0-no-copyleft-exception` that OSI does not seem to capture
* OSI not including the SPDX identifier `UCL-1.0` for the `Upstream Compatibility License v.1.0` 
Given that we have to generate this list for ourselves (for the CI and for an external webform), I thought I would include it here as well in case others want to build on these. 

And just to be clear, these are all and only OSI-approved licenses, it's just that there are some small inconsistencies in their listing.